examples/docs/package.json

```diff
@@ -9,8 +9,12 @@
     "preview": "astro preview"
   },
   "dependencies": {
+    "@algolia/client-search": "^4.12.0",
     "@docsearch/css": "^3.0.0",
-    "@docsearch/react": "^1.0.0-alpha.28"
+    "@docsearch/react": "^3.0.0",
```

Comment:
1.0 was asking for an outdated version of React, so I went ahead and pulled in the update to v3.0 that we'd already tested on the Astro docs site.